src/rp2_common/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ if (NOT PICO_BARE_METAL)
     pico_add_subdirectory(pico_mem_ops)
     pico_add_subdirectory(pico_malloc)
     pico_add_subdirectory(pico_printf)
+    pico_add_subdirectory(pico_bootsel_via_double_reset)
```

Comment:
I guess the ordering of things here isn't obvious, this really belongs before pico_multicore above